### PR TITLE
Qa 2281 create case table automation ids

### PR DESCRIPTION
**Pull request by bpierce01**
## Description
- Added automation IDs to cases summary view and cases table view

### CODE REVIEW

**Review by ManjunathJayanna — APPROVED** [review]

File: packages/portal-proto/src/components/Table/TotalItem.tsx [hunk 1]
```diff
@@ -32,7 +32,8 @@ const TotalItems: React.FC<TotalItemsProps> = ({
 
   return (
     <>
-      Total of <b>{formattedTotal}</b> {itemText}
+      Total of <b data-testid="text-total-item-count">{formattedTotal}</b>{" "}
```
Comment (amlehman) [inline packages/portal-proto/src/components/Table/TotalItem.tsx:35]: Having the test id on this nested bold tag is a little strange to me 🤔. We could move it up to the vertical table component [here](https://github.com/NCI-GDC/gdc-frontend-framework/blob/develop/packages/portal-proto/src/components/Table/TableHeader.tsx#L95) but I'm not sure if you're trying to specifically select the table headers with item counts.

File: packages/portal-proto/src/components/PopupIconButton/PopupIconButton.tsx [hunk 2]
```diff
@@ -26,6 +26,7 @@ export const PopupIconButton = forwardRef<
   ): JSX.Element => {
     return (
       <Button
+        data-testid={`Link to open ${label}`}
```
Comment (amlehman) [inline packages/portal-proto/src/components/PopupIconButton/PopupIconButton.tsx]: Can this id be the same format (kebab case) as the rest of the test ids?

**Review by amlehman — APPROVED** [review]

File: packages/portal-proto/src/components/PopupIconButton/PopupIconButton.tsx (hunk 2 above)
Comment (bpierce01) [reply packages/portal-proto/src/components/PopupIconButton/PopupIconButton.tsx]: Done! Updated the id to be correct case as suggested

File: packages/portal-proto/src/components/Table/TotalItem.tsx (hunk 1 above)
Comment (bpierce01) [reply packages/portal-proto/src/components/Table/TotalItem.tsx:35]: I put the ID there to have it directly on an item count we may check. If it doesn't hurt current or future formatting i'd like it there. Let me know

**Review by craigrbarnes — APPROVED** [review]

**Pull request merged**